docs/source/core_services/control/AuthenticationCommands.rst

```diff
@@ -26,20 +26,30 @@ All authentication sub-commands can be viewed by entering following command.
 
     subcommands:
         add                 add new authentication record
+        add-group           associate a group name with a set of roles
         add-known-host      add server public key to known-hosts file
+        add-role            associate a role name with a set of capabilities
         keypair             generate CurveMQ keys for encrypting VIP connections
```

Comment:
Can we print that keypair to standand out so that we can easily copy/paste it around